LocalReferenceLinkRel/explainer.md

```diff
@@ -116,16 +118,34 @@ files. Loop detection may be necessary for references to another
     }
   </style>
   <p>Inside Shadow DOM</p>
+    <template shadowrootmode="open">
+      <link rel="stylesheet" href="#inline_styles_from_shadow" />
+      <p>Inside Nested Shadow DOM</p>
+    </template>
 </template>
-<link rel="stylesheet" href="foo.css" id="#inline_styles_from_shadow" />
 <p>
-  Outside Shadow DOM. Styles defined inside the Shadow Root are not applied, so
-  this text is not blue.
+  Styles defined inside the parent Shadow Root are not applied, so "Inside Nested Shadow DOM" is not blue.
 </p>
 ```
 
-Due to existing Shadow DOM scoping behaviors, `<style>` tags defined inside the
-Shadow DOM cannot be accessed from the Light DOM.
+```html
+<template shadowrootmode="open">
+  <style id="inline_styles_from_shadow">
+    p {
+      color: blue;
+    }
+  </style>
+  <p>Inside Shadow DOM</p>
+</template>
+<template shadowrootmode="open">
+  <link rel="stylesheet" href="#inline_styles_from_shadow" />
+  <p>Inside Sibling Shadow DOM</p>
+</template>
+
```

Comment:
nit: the other example doesn't have a space here. Probably worth adding one there or removing here for consistency.

Although as Andy noted, we could have the <p></p> text live outside of the test case, in which case, this can be ignored

Reply:
Moved outside, per Andy's suggestion